src/plugins/intel_cpu/src/nodes/interaction.cpp

```diff
@@ -0,0 +1,229 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <algorithm>
+#include <chrono>
+#include <cmath>
+#include <string>
+#include <vector>
+
+#include "ie_parallel.hpp"
+#include "ngraph_transformations/op/interaction.hpp"
+#include "interaction.h"
+#include "utils/general_utils.h"
+#include <onednn/dnnl.h>
+#include <dnnl_extension_utils.h>
+#include <immintrin.h>
+#include "nodes/common/cpu_convert.h"
+#include "memory_desc/cpu_memory_desc_utils.h"
+#include "memory_desc/dnnl_blocked_memory_desc.h"
+#include "common/bfloat16.hpp"
+#include "common/cpu_memcpy.h"
+
+namespace ov {
+namespace intel_cpu {
+namespace node {
+
+Interaction::Interaction(const std::shared_ptr<ngraph::Node>& op, const dnnl::engine& eng, WeightsSharing::Ptr &cache)
+        : Node(op, eng, cache) {
+    std::string errorMessage;
+    if (isSupportedOperation(op, errorMessage)) {
+        errorPrefix = "Interaction node with name '" + getName() + "'";
+    } else {
+        IE_THROW(NotImplemented) << errorMessage;
+    }
+}
+
+void Interaction::initSupportedPrimitiveDescriptors() {
+    if (!supportedPrimitiveDescriptors.empty())
+        return;
+    dataPrecision = getOriginalInputPrecisionAtPort(0);
+    if (!one_of(dataPrecision, InferenceEngine::Precision::BF16, InferenceEngine::Precision::FP32))
+        IE_THROW() << errorPrefix << " has unsupported 'data' input precision: " << dataPrecision.name();
```

Comment:
Do not throw exception here. Set supported precision to PortConfigurator. Appropriate convert will be added automatically.

Reply:
fix in c67fb3cc606836770a72b723ec7843f237c2864a